web/app/scripts/src/directives/proportion-field.es.js

```diff
@@ -66,6 +66,16 @@ module.directive('proportionField', [
                         return;
                     }
 
+                    if ( Object.values(newModel).every( (x) => { return x === null; } ) ) {
+                        return;
+                    }
+
+                    for ( var [k, v] of Object.entries(newModel) ) {
```

Comment:
We normally try to use `forEach`: `Object.entries(newModel).forEach((k, v) => {...})`